cmd/clusterctl/client/move.go

```diff
@@ -63,7 +66,7 @@ func (c *clusterctlClient) Move(options MoveOptions) error {
 		options.Namespace = currentNamespace
 	}
 
-	if err := fromCluster.ObjectMover().Move(options.Namespace, toCluster); err != nil {
+	if err := fromCluster.ObjectMover().Move(options.Namespace, toCluster, options.DryRun); err != nil {
```

Comment:
In case of dry-run, is it possible to avoid to create the toCluster/runningEnsureCRSs on it?

Reply:
yes, let me think about it

Reply:
updated just now